packages/page-accounts/src/Sidebar/SubAccountsModal.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright 2017-2021 @polkadot/app-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { AccountId } from '@polkadot/types/interfaces';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { useTranslation } from '@polkadot/app-accounts/translate';
+import { AddressMini, Modal } from '@polkadot/react-components';
+
+interface Props {
+  className?: string;
+  onClose: () => void;
+  subs: AccountId[];
+}
+
+function SubAccountsModal ({ className, onClose, subs }: Props): React.ReactElement<Props> {
+  const { t } = useTranslation();
+
+  return (
+    <Modal
+      className={className}
+      header={t<string>('sub-identities')}
+      onClose={onClose}
+      size='tiny'
+    >
+      <Modal.Content className='content'>
```

Comment:
It would be perfect to have just the content scrollable, with the header pinned to top. But perhaps that's a separate improvement to the new modal.

Reply:
I've set only content to be scrollable but only in this modal. But if it's needed we can make another pr with setting this to all modals.